commit 6f8222a1c52a9e577925b90d16b95be10ef50596
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Tue Apr 19 23:27:27 2016 +0200

    Fix gprof timing
    
    	* sysdeps/mach/hurd/profil.c (__profile_frequency): Return tick
    	frequency instead of tick length in us.

diff --git a/sysdeps/mach/hurd/profil.c b/sysdeps/mach/hurd/profil.c
index be249bb..a2c4e55 100644
--- a/sysdeps/mach/hurd/profil.c
+++ b/sysdeps/mach/hurd/profil.c
@@ -102,7 +102,7 @@ update_waiter (u_short *sample_buffer, size_t size, size_t offset, u_int scale)
 int
 __profile_frequency (void)
 {
-  return profile_tick;
+  return 1000000 / profile_tick;
 }
 libc_hidden_def (__profile_frequency)
 
